src/components/sections/Footer.svelte

```diff
@@ -0,0 +1,28 @@
+<style>
+.main-footer {
+  grid-column: main;
+  display: flex;
+  justify-content: flex-end;
+  width: var(--main-content-width);
+  margin: var(--space-2x) auto;
+}
+.project-links {
+  list-style: none;
+  display: grid;
+  font-size: var(--text-015);
+  grid-gap: var(--space-2x);
+  grid-template-columns: auto auto auto;
+  padding: calc(var(--space-2x) * 0.7) var(--space-2x);
```

Comment:
```suggestion
  padding: var(--space-base) var(--space-4x);
```

Reply:
Feel free to take or leave this suggestion:

(1) the 4x for the left/right helps it breath a little more against the curve of the body element. 
(2) `space-base` keeps the footer equal to the space from top of app  bar to the top of the body element